pkg/ast/expr.go

```diff
@@ -26,6 +31,7 @@ type NameNode interface {
 type Expr interface {
 	Node
 	expr()
+	String() string
```

Comment:
I think `String` is not clear to describe what this function does，plz also add the comment